demosplan/DemosPlanCoreBundle/ResourceTypes/RoleResourceType.php

```diff
@@ -54,7 +54,7 @@ public function isGetAllowed(): bool
 
     public function isListAllowed(): bool
     {
-        return false;
+        return $this->isAvailable();
```

Comment:
I would write true directly

Reply:
Why? In every resourcetype we use this method for true

Reply:
i think this should be something like return $this->currentUser->hasPermission('feature_user_list');

Reply:
this permission allows the current user to fetch all users the current user is allowed to see from the JSON:API route, it's seems to be enabled everywhere (most of time for the support Role), but i'm honestly nt 100% sure, so it make sense for me to use here to check when the list should be allowed or not, may be u can check it too just to be sure

Reply:
done in [8b33081](https://github.com/demos-europe/demosplan-core/pull/3475/commits/8b33081bee61d3594962747f4e2bacfbfa411568)

Reply:
sorry my bad i thought this is the user resource type @sierrazamudiodemos u re right it works for the support role so it has to be a permissions problem